app/helpers/self_service/breadcrumbs_helper.rb

```diff
@@ -62,5 +66,9 @@ def crumb_firm_adviser_new
     def crumb_trading_name_new
       { name: t('self_service.trading_name_new.breadcrumb', firm_name: @firm.registered_name) }
     end
+
+    def crumb_principal_edit
+      { name: t('self_service.principal_edit.breadcrumb', firm_name: @firm.registered_name) }
+    end
   end
 end
```

Comment:
I was just checking to see if there was value in adding tests. It looks like there's a bit more than just configuration going on in there (even if that is the end result), but I don't have the context behind this. If you're happy that no specs are required for this then I shall :+1: :)